test/e2e/run_test.go

```diff
@@ -1567,4 +1567,29 @@ WORKDIR /madethis`
 		Expect(session.OutputToString()).To(ContainSubstring("mysecret"))
 
 	})
+
+	It("podman run --dep", func() {
+		depName := "ctr1"
+		depContainer := podmanTest.Podman([]string{"create", "--name", depName, ALPINE, "top"})
+		depContainer.WaitWithDefaultTimeout()
+		Expect(depContainer.ExitCode()).To(Equal(0))
```

Comment:
maybe not in this test, but it would be good to have 2 or 3 dependency containers.